### PR TITLE
Update travis to test more versions of node

### PR DESCRIPTION
**Pull request by LionOps**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,16 @@
-sudo: false
```
Comment (adrianmoisey) [inline .travis.yml:1]: Why was this removed? Having it there makes Travis use their container based build system, which is faster then the regular one.

Comment (adrianmoisey) [inline .travis.yml:1]: (details here: https://docs.travis-ci.com/user/workers/container-based-infrastructure/)

Comment (LionOps) [inline .travis.yml:1]: I thought it was the default. In looking at the documentation, it is the default for projects that began using Travis on 1/1/2015.

It's unneeded for this project. Should we include it anyway, though?

Comment (adrianmoisey) [inline .travis.yml:1]: > it is the default for projects that began using Travis on 1/1/2015.

Oh wow, I didn't know that.

> It's unneeded for this project. Should we include it anyway, though?

Nah, leave it out then.

**Comment by adrianmoisey** [comment]
:+1:
